plugin/api.go

```diff
@@ -348,6 +348,11 @@ type API interface {
 	// Minimum server version: 5.2
 	GetChannel(channelId string) (*model.Channel, *model.AppError)
 
+	// GetChannels gets channels by options
+	//
+	// Minimum server version: 5.18
+	GetChannels(options *model.GetChannelsOptions) (*model.ChannelList, *model.AppError)
```

Comment:
2/5 that we should pass the struct directly here. This would make it more clear, that this field is needed.